### PR TITLE
Copy button update

### LINKED ISSUE #158: Create Icon Only Variant of CopyButton
Since the `CopyButton` is based off of the standard `Button` component, it would make sense to have an easier way to produce an icon only variant of the `CopyButton`. Currently you can mimic some of this behavior by passing in and empty string for the `buttonText` prop, but ideally there should be a dedicated prop for this behavior.

### PR DESCRIPTION
**Pull request by NehalBhandariSlalom**
This PR creates an IconOnly prop to set the IconOnly property. If set to true, it will display icon only. If approved this will close #158.
Screen Shot 2022-02-01 at 3 02 39 PM

### CODE REVIEW

**Review by adorack-tri — APPROVED** [review]
It might be helpful to update StoryBook to show off this additional feature.

File: src/CopyButton/CopyButton.tsx [hunk 1]
```diff
@@ -28,6 +28,11 @@ export interface CopyButtonProps {
      * The value to be copied to the clipboard.
      */
     valueToCopy: string;
+    /**
+     * This to set the iconOnly property. If set to false, the icon and text will appear. If true, only copy icon will render.
```
Comment (jason-ransom-slalom) [inline src/CopyButton/CopyButton.tsx:32]: minor: Should read "This is to set..."

**Review by amirkibbar — APPROVED** [review]

**Review by jason-ransom-slalom — APPROVED** [review]
Approved with minor comment.

**Comment by NehalBhandariSlalom** [comment]
> It might be helpful to update StoryBook to show off this additional feature.

In the storybook, under controls we can toggle the iCon Button to show or hide the text. Are you looking for any specific functionality here?

**Comment by adorack-tri** [comment]
> In the storybook, under controls we can toggle the iCon Button to show or hide the text. Are you looking for any specific functionality here?

Never mind; I was thinking of a different type of property where you needed to edit JSON to toggle things.  I think it's fine!

**Pull request merged**